docs/sql-guide/functions/function-str.md

```diff
@@ -0,0 +1,88 @@
+---
+title: STR()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# STR() function
+
+`STR()` function returns the given number formatted to a String of a certain length, with given precision.
+
+## Syntax
+
+```
+STR(num_expr, lenofstr, precision)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type | Return value |
+|---|---|---|---|
+| `num_expr` | Required `int` or `float` value to format as a `string`. |
+| `lenofstr` | Optional `int` value for the length of the output string. Defaults to 10. |
+| `precision` | Optional `int` value for the precision to round `num_expr` to. Defaults to 0. |
+
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `string` | Returns a string representation of `num_expr` rounded to the `precision` and space-padded to the `length`. |
```

Comment:
space padded to the left or right? Let's be specific. Examples seem to indicate left padded?

Reply:
Left padded, added!